.github/workflows/ci.yaml

```diff
@@ -28,6 +28,11 @@ on:
         description: "(used if BUMP_MANIFEST=true) If true: attempt to PR/merge manifest branch"
         default: false
         required: false
+      CUDA_IMAGE:
+        type: string
+        description: CUDA image to use as base, e.g. nvidia/cuda:X.Y.Z-devel-ubuntu22.04
+        default: 'latest'
```

Comment:
Either you need to add special treatment of the "latest" value, or put the actual default value here instead of in the Dockerfile (https://github.com/NVIDIA/JAX-Toolbox/blob/400d83af7df9f93a5271d7f0c8daab7a47278ae8/.github/container/Dockerfile.base#L2)

Reply:
The logic was already in place from a long time ago: https://github.com/NVIDIA/JAX-Toolbox/blob/123e033ae794a5f1dbac566885d0fc2441cfb2ca/.github/workflows/_build_base.yaml#L135
😁 

It was also the conclusion from a previous discussion that we should not repeat the default value multiple times in the JAX-Toolbox code base. Placing it in the Dockerfile is the only option so that:
1. we have a single occurrence of this default value, while still make sure
2. the dockerfile is self-contained.

Reply:
Wait. I just realized that the logic is somehow broken. Working on a fix.

Reply:
Fixed it? =)